src/utils/common.js

```diff
@@ -595,3 +595,8 @@ export const formatBigFiatAmount = (value: Value, fiatCurrency: string) => {
 export const getEnsName = (username: string) => `${username}${getEnsPrefix()}`;
 
 export const extractUsernameFromEnsName = (ensName: string) => ensName.replace(getEnsPrefix(), '');
+
+export const valueAsKey = (value: string | number): string => {
```

Comment:
```suggestion
export const addressAsKey = (value: string): string => {
```

Other potential names to consider `toAddressKey`, `keyFromAddress`, etc

Naming should reflect that this is intended for address keys, I think that we do not want to assume that each record value needs to be lowercase. wdyt?

Reply:
Honestly I would prefer this for more generic name, but we use it for address only as for now so I guess it's ok, replaced